src/wp-includes/css/custom-properties.css

```diff
@@ -64,6 +69,11 @@ body {
 	--wp-admin--input--background--readonly: #f0f0f1;
 	--wp-admin--checkbox--fill: var(--wp-admin--button-primary);
 
+	--wp-admin--radio--color--focus: #1d2327;
+	--wp-admin--radio--border--checked: #8c8f94;
+	--wp-admin--radio--border--focus: #4f94d4;
+	--wp-admin--radio--box-shadow: var(--wp-admin--themes--blue);
```

Comment:
These aren't radio controls, it's for the buttons here:

<img width="206" alt="Screen Shot 2021-10-21 at 5 34 27 PM" src="https://user-images.githubusercontent.com/541093/138360159-6af152c3-1601-4995-aeca-6f5b5a147bd3.png">

I think these properties should be either merged with button properties (if they exist), or change the name to use `button` for the location.

Reply:
I agree - leaving them named as radio in either case would be confusing, case in point being that I couldn't locate that control you brought up and thought they were a hidden / optional _radio_ button based on the property name.